appsvc-fnc-dev-gcxonboardsync-dotnet001/OnboardUsers.cs

```diff
@@ -268,11 +281,94 @@ private static async Task<List<string>> GetUsersToOnboard(GraphServiceClient gra
             return userIds;
         }
 
-        private static async Task<string> GetWelcomeGroupId(GraphServiceClient graphAPIAuth, string welcomeGroupIds, int UserCount, int WelcomeGroupMemberLimit, ILogger log)
+        //private static async Task<string> GetWelcomeGroupId(GraphServiceClient graphAPIAuth, string welcomeGroupIds, int UserCount, int WelcomeGroupMemberLimit, ILogger log)
+        //{
```

Comment:
If this is not need anymore. Cade can be deleted. If we need the code back, it will be part of github history.